e2e/framework/manager/rook_test_infra.go

```diff
@@ -34,7 +34,7 @@ var (
 )
 
 const (
-	temp_image_file_name           = "temp_image.tar"
+	tempImageFileName              = "temp_image.tar"
```

Comment:
in general we would want to limit our changes in a PR to the topic of the PR. For example, the cleanup changes would go in their own PR rather than being included in this one.